### PR TITLE
rename enabled staking to sybil protection enabled

### PR DESCRIPTION
**Pull request by ceyonur**
Renamed staking enabled to sybil protection enabled to clean any confusions due to word "staking"

### CODE REVIEW

File: chains/manager.go [hunk 1]
```diff
@@ -166,10 +166,10 @@ type ChainConfig struct {
 }
 
 type ManagerConfig struct {
-	StakingEnabled bool            // True iff the network has staking enabled
-	StakingCert    tls.Certificate // needed to sign snowman++ blocks
-	StakingBLSKey  *bls.SecretKey
-	TracingEnabled bool
+	SybilProtectionEnabled bool            // True iff the network has staking enabled
```
Comment (abi87) [inline chains/manager.go:169]: nit: comment seems outdated and frankly not really informative. Consider dropping it

File: config/flags.go [hunk 2]
```diff
@@ -267,7 +267,6 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Bool(StakingEphemeralSignerEnabledKey, false, "If true, the node uses an ephemeral staking signer key")
 	fs.String(StakingSignerKeyPathKey, defaultStakingSignerKeyPath, fmt.Sprintf("Path to the signer private key for staking. Ignored if %s is specified", StakingSignerKeyContentKey))
 	fs.String(StakingSignerKeyContentKey, "", "Specifies base64 encoded signer private key for staking")
-
 	// TODO: Remove this flag in the future
 	fs.Uint64(StakingDisabledWeightKey, 100, "Weight to provide to each peer when staking is disabled")
```
Comment (abi87) [inline config/flags.go:271]: q: should `StakingDisabledWeightKey` become `SybilProtectionDisabledWeightKey`?

File: node/node.go [hunk 3]
```diff
@@ -254,13 +254,13 @@ func (n *Node) initNetworking(primaryNetVdrs validators.Set) error {
...
-	n.Config.BenchlistConfig.StakingEnabled = n.Config.EnableStaking
+	n.Config.BenchlistConfig.SybilProtectionEnabled = n.Config.SybilProtectionEnabled
 	n.benchlistManager = benchlist.NewManager(&n.Config.BenchlistConfig)
 
 	n.uptimeCalculator = uptime.NewLockedCalculator()
 
 	consensusRouter := n.Config.ConsensusRouter
-	if !n.Config.EnableStaking {
+	if !n.Config.SybilProtectionEnabled {
 		// Staking is disabled so we don't have a txID that added us as a
```
Comment (abi87) [inline node/node.go:264]: nit: I'd slighly reword the comment, replacing staking with sybil protection

File: node/node.go [hunk 4]
```diff
@@ -767,7 +767,7 @@ func (n *Node) initVMs() error {
 	// If staking is disabled, ignore updates to Subnets' validator sets
```
Comment (abi87) [inline node/node.go:767]: nit: again here in the comment, I'd replace staking with sybil protection

**Review by abi87 — DISMISSED** [review]
found nits around comments and a question

File: chains/manager.go (hunk 1 above)
Comment (ceyonur) [reply chains/manager.go]: done

File: config/flags.go (hunk 2 above)
Comment (ceyonur) [reply config/flags.go:271]: we deprecated that in #1370, will remove it in the future.

File: node/node.go (hunk 3 above)
Comment (ceyonur) [reply node/node.go]: good catch, renamed

File: node/node.go (hunk 4 above)
Comment (ceyonur) [reply node/node.go]: renamed

**Review by danlaine — APPROVED** [review]

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**